### ISSUE TITLE
Is it working, README

### ISSUE THREAD

**Issue by omidb** [stmharry/DeepBox#1]
Hi,

It seems that you have the implementation of DeepBox here, is it working? do you have any trained model? would you please add a minimal instruction to run this?

Thanks

**Comment by stmharry** [comment]
@omidb 

Yes, it works, and I currently use DeepBox as the base for some other projects. The thing is I am constantly modifying it to conform to new projects so that it is not always backward compatible. I will put this to my todo list but no guarantees :P

Thank you for kindly reminding me that I still have to maintain this package!
